tests/fail/data_race/read_write_race.rs

```diff
@@ -13,9 +13,7 @@ pub fn main() {
     let b = &mut a as *mut u32;
     let c = EvilSend(b);
     unsafe {
-        let j1 = spawn(move || {
-            *c.0
```

Comment:
If we add a trailing `;` here, does rustfmt keep the braces? I think I'd prefer that, for symmetry with the other `spawn`.